make test fails in Mac

I ran `make test` on Mac and it showed an error due to a difference in the `sam_summary` files:

```
diff -qs <(xzcat output/reads_1___reads_2___reads_3___reads_4.sam_summary.xz) <(xzcat data/reads_1___reads_2___reads_3___reads_4.sam_summary.xz)
Files /dev/fd/63 and /dev/fd/62 differ
make: *** [test] Error 1
```

The difference between the files seems to only be in the position of a few hits. (Seems to be calling it 8191 bp away in SAMN07333007.contig00001)

```
3705,3706c3705,3706
< SAMEA3889837.contig00631-1	16	SAMN07333007.contig00001	1682717	60	150=	*	0	0	ATAACTGGAATATACTTTCGTATATACGTAACTACAAGCACAATCAAAACAATGCCAACGATTGTCAGGAACATATTCCAAAAACTGAAATAATCCATAACCATCGGTATAACGGAATCCATATGACCAACCTGTGTGAAATCAGACGGA	*	NM:i:0	ms:i:300	AS:i:300	nn:i:0	tp:A:P	cm:i:23	s1:i:135	s2:i:0	de:f:0	rl:i:0
< SAMN03010432.contig00007-1	16	SAMN07333007.contig00001	1524412	60	150=	*	0	0	CCTGGATCTACATCGTCATAAAGACCAGCATAAGATACAGAGAACTTCTCTGCAGTACCTAATACTTGCTCAGTAGAAAGAACTACTTCTGCACCTGTTACAAGCTCAGCTTGTCCGTCTACGAAGTCATGAGTACGGATTTCTGGACCT	*	NM:i:0	ms:i:300	AS:i:300	nn:i:0	tp:A:P	cm:i:17	s1:i:133	s2:i:0	de:f:0	rl:i:0
---
> SAMEA3889837.contig00631-1	16	SAMN07333007.contig00001	1690908	60	150=	*	0	0	ATAACTGGAATATACTTTCGTATATACGTAACTACAAGCACAATCAAAACAATGCCAACGATTGTCAGGAACATATTCCAAAAACTGAAATAATCCATAACCATCGGTATAACGGAATCCATATGACCAACCTGTGTGAAATCAGACGGA	*	NM:i:0	ms:i:300	AS:i:300	nn:i:0	tp:A:P	cm:i:23	s1:i:135	s2:i:0	de:f:0	rl:i:0
> SAMN03010432.contig00007-1	16	SAMN07333007.contig00001	1532603	60	150=	*	0	0	CCTGGATCTACATCGTCATAAAGACCAGCATAAGATACAGAGAACTTCTCTGCAGTACCTAATACTTGCTCAGTAGAAAGAACTACTTCTGCACCTGTTACAAGCTCAGCTTGTCCGTCTACGAAGTCATGAGTACGGATTTCTGGACCT	*	NM:i:0	ms:i:300	AS:i:300	nn:i:0	tp:A:P	cm:i:17	s1:i:133	s2:i:0	de:f:0	rl:i:0
3709c3709
< SAMN07332871.contig00006-1	16	SAMN07333007.contig00001	1593707	60	150=	*	0	0	TGTTCGTATCATTCTTTTGAACAGTAGCACTAGCAATATACGTACCATTTTTTAAAGAACTTGTACTTTCTTGCTTAAAGATAGCAAACGTAATTGGGGATTTATTATACTTGTCATTTTTATGAATAGCTTCAATCAGTTGATTGGCAA	*	NM:i:0	ms:i:300	AS:i:300	nn:i:0	tp:A:P	cm:i:22	s1:i:137	s2:i:0	de:f:0	rl:i:0
---
> SAMN07332871.contig00006-1	16	SAMN07333007.contig00001	1601898	60	150=	*	0	0	TGTTCGTATCATTCTTTTGAACAGTAGCACTAGCAATATACGTACCATTTTTTAAAGAACTTGTACTTTCTTGCTTAAAGATAGCAAACGTAATTGGGGATTTATTATACTTGTCATTTTTATGAATAGCTTCAATCAGTTGATTGGCAA	*	NM:i:0	ms:i:300	AS:i:300	nn:i:0	tp:A:P	cm:i:22	s1:i:137	s2:i:0	de:f:0	rl:i:0
```

--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ test: ## Run everything but just with 3 batches to test full pipeline
 	snakemake $(SMK_PARAMS) -j 99999 --config batches=data/batches_small.txt -- download  # download is not benchmarked
 	scripts/benchmark.py --log logs/benchmarks/test_match_$(DATETIME).txt "snakemake $(SMK_PARAMS) --config batches=data/batches_small.txt nb_best_hits=1 -- match"
 	scripts/benchmark.py --log logs/benchmarks/test_map_$(DATETIME).txt   "snakemake $(SMK_PARAMS) --config batches=data/batches_small.txt nb_best_hits=1 -- map"
-	diff -qs <(xzcat output/reads_1___reads_2___reads_3___reads_4.sam_summary.xz | cut -f -3) <(xzcat data/reads_1___reads_2___reads_3___reads_4.sam_summary.xz  | cut -f -3)
+	diff -s <(xzcat output/reads_1___reads_2___reads_3___reads_4.sam_summary.xz | cut -f -3) <(xzcat data/reads_1___reads_2___reads_3___reads_4.sam_summary.xz  | cut -f -3)
 
 download: ## Download the 661k assemblies and COBS indexes, not benchmarked
 	snakemake $(SMK_PARAMS) -j 99999 -- download

